tests/e2e/__testers__.ts

```diff
@@ -92,60 +201,64 @@ export const testTemplate = (params: {
     /**
      * Drop database before running the tests case it wasn't cleaned up from the previous test run.
      */
+
     await ctx.dropTestDatabase()
+    await ctx.createTestDatabase()
+
+    const initResult = await ctx.runAsync(`npm run init`, { reject: true })
 
-    /**
-     * Test 1
-     * Check that the initialization script works. This includes running migrate triggering generators and executing the seed.
-     */
-    const initResult = ctx.run(`npm run init`, { reject: true })
     expect(initResult.stderr).toMatch('')
     expect(stripAnsi(initResult.stdout)).toMatch('Generated Prisma Client')
     expect(stripAnsi(initResult.stdout)).toMatch('Running seed command')
 
-    const prisma = await ctx.getPrisma()
-    const prismaAdmin = await ctx.getPrismaAdmin()
+    // TODO Test the Vercel API (next dev for Blog template)
+    // await ctx.fs.writeAsync('.vercel/project.json', {
+    //   projectId: 'prj_6yrTe9CGQagAQwGjr7JEejkxhz3A',
+    //   orgId: 'team_ASKXQ5Yc1an2RqJc5BCI9rGw',
+    // })
+  })
 
-    try {
-      /**
-       * Test 2
-       * Check that the template migration script works.
-       */
+  /**
+   * Test 2
+   * Check that the template migration script works.
+   */
+  if (params.templateName !== 'Empty') {
+    it(`${params.templateName} - template migration script should work`, async () => {
       await ctx.dropTestDatabase()
-      await prismaAdmin.$executeRawUnsafe(`create database ${ctx.databaseName}`)
+      await ctx.createTestDatabase()
+
+      console.log('Get getPrisma()')
+      const prisma = await ctx.getPrisma()
       await Reflector.Client.runMigrationScript(
         prisma,
         ctx.template.migrationScript,
-        // TODO test a matrix of data sources
-        'postgres'
+        params.datasourceProvider
       )
-
-      /**
-       * Test 3
-       * Check the seed again but this time using the derived seed function.
-       */
+    })
+  }
+  /**
+   * Test 3
+   * Check the seed again but this time using the derived seed function.
+   */
+  if (params.templateName !== 'Empty') {
+    it.skip(`${params.templateName} - seed using the derived seed function should work`, async () => {
+      const prisma = await ctx.getPrisma()
```

Comment:
Since we have two prisma clients, we can probably improve the previous codebase by have this function for example called `getApplicationPrisma` vs `getAdminPrisma`.